### PR TITLE
Query CLI & gRPC for Standard Price Reference

### PR DESCRIPTION
**Pull request by ntchjb**
Fixed: [BAND-241](https://bandprotocol.atlassian.net/browse/BAND-241)

## Implementation details

This PR implement query CLI command and gRPC to query latest prices from standard price reference's oracle script. Every time when there is a new oracle request requesting data from standard price reference oracle script, the result will be stored in a local leveldb database, so that we can query the latest price from it.

Since every ways we send query to Bandchain (gRPC, REST of gRPC-gateway, CLI), all queries will be processed at ABCI's `Query()`. Therefore, new hook has been created to `ApplyQuery()` before the baseapp's `Query()` to perform custom query. In this case, we perform querying from our local leveldb.

## Please ensure the following requirements are met before submitting a pull request:

- [ ] The pull request is targeted against the correct target branch
- [ ] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [ ] The pull request includes a description of the implementation/work done in detail.
- [ ] The pull request includes any and all appropriate unit/integration tests
- [ ] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [ ] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

File: proto/oracle/v1/query.proto [hunk 1]
```diff
@@ -256,14 +256,22 @@ message QueryRequestPriceRequest {
 // QueryRequestPriceResponse is response type for the Query/RequestPrice RPC
 // method.
 message QueryRequestPriceResponse {
```
Comment (taobun) [inline proto/oracle/v1/query.proto:258]: Request must be a list of symbols?

Comment (taobun) [reply proto/oracle/v1/query.proto:258]: image

Comment (taobun) [reply proto/oracle/v1/query.proto:258]: And response must be a list of Price struct

**Review by taobun — CHANGES_REQUESTED** [review]

File: proto/oracle/v1/query.proto (hunk 1 above)
Comment (ntchjb) [reply proto/oracle/v1/query.proto:258]: Fixed, but response will be an object with a field, which is a list of price results.

File: hooks/price/types.go [hunk 2]
```diff
@@ -1,30 +1,10 @@
 package price
```
Comment (taobun) [inline hooks/price/types.go:1]: Cam we delete this file?

**Review by taobun — CHANGES_REQUESTED** [review]

File: hooks/price/types.go (hunk 2 above)
Comment (ntchjb) [reply hooks/price/types.go:1]: Actually, `Input` and `Output` are used in `AfterEndBlock` of `hooks/price/price.go` to decode result's calldata and result.

Comment (taobun) [reply hooks/price/types.go:1]: I see

**Review by taobun — APPROVED** [review]

**Review by songwongtp — APPROVED** [review]

**Pull request merged**